From 4c0c5378d032f2f95577585935624baf7b4decf3 Mon Sep 17 00:00:00 2001
From: Yu Chien Peter Lin <peterlin@andestech.com>
Date: Wed, 5 Jan 2022 11:02:26 +0800
Subject: [PATCH] Prevent fw_dynamic from relocation

This patch prevents OpenSBI relocation, load fw_dynamic to link address

Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
---
 board/AndesTech/ax25-ae350/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/AndesTech/ax25-ae350/Kconfig b/board/AndesTech/ax25-ae350/Kconfig
index e50f505a..385c4c11 100644
--- a/board/AndesTech/ax25-ae350/Kconfig
+++ b/board/AndesTech/ax25-ae350/Kconfig
@@ -25,7 +25,7 @@ config SPL_TEXT_BASE
	default 0x800000

 config SPL_OPENSBI_LOAD_ADDR
-	default 0x01000000
+	default 0x0

 config BOARD_SPECIFIC_OPTIONS # dummy
	def_bool y
--
2.25.1
